src/style.css

```diff
@@ -16,6 +16,31 @@ body {
   overflow-x: hidden;
 }
 
+/* nav */
+
+.nav-burger-toggle {
+    background: transparent;
+    border: 0;
+}
+
+.nav-burger-line {
+    width: 30px;
+    height: 3px;
+    background-color: black;
+    margin: 3px 0;
+}
```

Comment:
Could we rewrite while `.nav-burger-toggle` and partly `.nav-burger-line` selector rules with bootstrap?)

Reply:
Pushed the fix.